environments/reana-demo-bsm-search/code/generantuple.py

```diff
@@ -0,0 +1,76 @@
+# import ROOT
```

Comment:
Shouldn't be necessary to copy or move code under environment, that would be also confusing to people. Note that we the `docker build` instruction to invoke mentioned in `.travis.yml`, it should set directories right.  (The README is wrong.)  So the user files don't have to be moved or copied.